packages/cli/src/commands/init/init.ts

```diff
@@ -453,37 +418,27 @@ async function createTemplateUri(
     return options.template;
   }
 
-  // Figure out whether we should use the @react-native-community/template over react-native/template,
-  // this requires 2 tests.
-
-  // Test #1: Does the @react-native-community/template@ exist?
-  const canUseCommunityTemplate = await npmCheckPackageVersionExists(
-    TEMPLATE_PACKAGE_COMMUNITY,
-    version,
-  );
-
-  // Test #2: Does the react-native@version package *not* have a template embedded. We know that
-  //          this applies to all version before 0.75. The 1st release candidate is the minimal
-  //          version that has no template.
+  // Does the react-native@version package *not* have a template embedded. We know that this applies to
+  // all version before 0.75. The 1st release candidate is the minimal version that has no template.
   const useLegacyTemplate = semver.lt(
     version,
     TEMPLATE_COMMUNITY_REACT_NATIVE_VERSION,
```

Comment:
This is `0.75.0-rc.0` so I'm unsure how this will work out with `lt`.
Should we just instead compare the minor from semver and do `<= 74`?